src/classes/CRLP_RollupProcessor.cls

```diff
@@ -543,6 +553,25 @@ public inherited sharing class CRLP_RollupProcessor {
         }
     }
 
+    /**
+     * @description Set needsUpdate and CustomizableRollups_UseSkewMode__c true if >= Skew Mode Threshold
+     * @param updatedRecord Update SObject record for rollup totals
+     * @param needsUpdate Boolean indicating whether or not record requires update
+     * @return Boolean
+     */
+    private Boolean needsSkewModeOverrideSet(SObject updatedRecord, Boolean needsUpdate) {
+        if (((Decimal) updatedRecord.get('npo02__NumberOfClosedOpps__c') >= maxRelatedOppsForNonLDVMode ||
```

Comment:
It's been a while, but if `npo02__NumberOfClosedOpps__c` is null, does using the (Decimal) typecast thrown an NPE? I seem to remember that used to be the case at one point, but I'm unsure if that's still true.